index.html

```diff
@@ -1,10 +1,76 @@
+    <div><a href="#Основне">Основне</a></div>
+    <div><a href="#Мови">Знання мов</a></div>
+    <div><a href="#Навички">Навички</a></div>
+    <div><a href="#Освіта">Освіта</a></div>
+</div>
+<div class="grid__container">
+    <div class="left__container">
+        <h2 id="Основне">Основна інформація</h2>
+        <p>Народилася в місті Черкаси. Закінчила 9 класів Черкаської спеціалізованої школи І-ІІІ ст. № 17. З 2016 року навчається в Черкаському державному бізнес-коледжі. Досвіду роботи немає</p>
+        <h2 id="Освіта">Освіта</h2>
+        <ul>
+            <li><b>2016 - н.ч.</b> — <a href="http://business-college.com.ua/">ЧДБК</a></li>
+            <li><b>2007 - 2016</b> — <a href="http://www.school17.ck.ua/">Черкаська СПШ №17</a></li>
+            <li><b>2006 - 2015</b> — <a href="http://music-school-1.ck.ua/">Черкаська ДМШ №1</a></li>
+        </ul>
+        <h2 id="Мови">Знання мов</h2>
+        <ul>
+            <li>українська — рідна;</li>
+            <li>російська — C2;</li>
+            <li>англійська — B2;</li>
+            <li>німецька — A1.</li>
+        </ul>
+        <h2>Особиста характеристика</h2>
+        <p>Люблю програмування, прослуховування/створення музики, дивитися фільми/мультфільми/серіали/мультсеріали, ігри, вікі і художні фентезі-книги.</p>
+        <p>Спокійна, врівноважена, дещо конфліктна, з поганим почуттям гумору.</p>
+    </div>
+    <div class="right__container">
+        <img class="avatar" src="img/avatar.svg"/>
+        <h1>Прокопчук Богдана Юріївна</h1>
+        <h4>22.06.2001</h4>
+        <h2 id="Навички">Навички</h2>
+        <div class="skillz">
+            <div style="background-color: rgba(255, 255, 255, 0.2)">
+                <h3>Python</h3>
+                <ul>
+                    <li>Основи</li>
+                    <li>Створення ботів для месенджерів за допомогою бібліотек</li>
+                    <li>Створення ВН за допомогою бібліотек</li>
+                </ul>
+            </div>
+            <div style="background-color: rgba(255, 255, 255, 0.4)">
+                <h3>HTML + CSS</h3>
+                <ul>
+                    <li>Основи верстки</li>
+                </ul>
+            </div>
+            <div style="background-color: rgba(255, 255, 255, 0.6)">
+                <h3>Інше</h3>
+                <ul>
+                    <li>Досвід роботи за їжу</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</div>
+<footer>
+    <a href="https://www.facebook.com/profile.php?id=100013932006432"><img class="sm" src="img/sm/Facebook.png"/></a>
+    <a href="https://www.instagram.com/socthecat/"><img class="sm" src="img/sm/Instagram.png"/></a>
+    <a href="https://soundcloud.com/sociochlorine"><img class="sm" src="img/sm/Soundcloud.png"/></a>
+    <a href="https://youtube.com/user/SocTheMad"><img class="sm" src="img/sm/YouTube.png"/></a>
+    <p style="font-size: 12px">© Богдана Прокопчук, 2015</p>
+</footer>
 </body>
 </html>
```

Comment:
live empty line in the end of the file. Setup your IDE to add it on save by default